apps/meteor/client/views/marketplace/AppDetailsPage/tabs/AppRequests/AppRequestItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { Badge, Box } from '@rocket.chat/fuselage';
+import { usePermission } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
+import React from 'react';
+
+import UserAvatar from '../../../../../components/avatar/UserAvatar';
+import { useFormatDateAndTime } from '../../../../../hooks/useFormatDateAndTime';
+
+type AppRequestItemProps = {
+	seen: boolean;
+	name: string;
+	createdDate: string;
+	message: string;
+	username: string;
+};
+
+const AppRequestItem = ({ seen, name, createdDate, message, username }: AppRequestItemProps): ReactElement => {
+	const formatDateAndTime = useFormatDateAndTime();
+	const isAdminUser = usePermission('manage-apps');
+
+	return (
+		<Box display='flex' flexDirection='row' pb='x12' pie='x24' mbe='x9' flexGrow='1'>
```

Comment:
what did we talk about overuse of boxing? at a minimum these components should be isolated


x9? it doesn't even exist (not like spacing token)

Reply:
This was probably a typo 😅 the correct number is x8. Fixed.